developer-tools.md

```diff
@@ -48,6 +48,24 @@ builds. This process will auto-start after the first time `build/mvn` is called
 shut down at any time by running `build/zinc-<version>/bin/zinc -shutdown` and will automatically
 restart whenever `build/mvn` is called.
 
+<h3>Building submodules individually</h3>
+
+For instance, you can build the Spark Core module using:
+
+```
+$ # sbt
+$ build/sbt
+> project core
+> package
+
+$ # or you can build the spark-core module with sbt directly using:
+$ build/sbt core/package
+
+$ # Maven
+$ build/mvn package -DskipTests -pl :spark-core_2.11
```

Comment:
This can be `-pl core`. Both work but maybe it's a little more consistent with the existing example and the one above. I think you can just show one sbt invocation, the second one, which is consistent with the existing one. They're both "building with sbt".